package.json

```diff
@@ -2,7 +2,7 @@
     "name": "vscode-azurefunctions",
     "displayName": "Azure Functions",
     "description": "%azureFunctions.description%",
-    "version": "1.8.3-alpha.0",
+    "version": "1.8.4-alpha.0",
```

Comment:
Wait, huh?

Reply:
What happened?

Reply:
I dunno, the bot bumped it twice I guess.

Reply:
It was done manually here, https://github.com/microsoft/vscode-azurefunctions/pull/3389/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519.

Did we make a release from a non-main branch?

Reply:
Yeah, I think Phillip and I both did two separate release branches